<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>挂号医生</string>
    </property>
    <addaction name="appointAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>看诊医生</string>
    </property>
    <addaction name="hendleAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>药房医生</string>
    </property>
    <addaction name="drugAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>缴费界面</string>
    </property>
    <addaction name="chargeAction"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>信息管理</string>
    </property>
    <addaction name="infoAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="appointAction">
   <property name="text">
    <string>挂号界面</string>
   </property>
  </action>
  <action name="hendleAction">
   <property name="text">
    <string>看诊界面</string>
   </property>
  </action>
  <action name="drugAction">
   <property name="text">
    <string>药房界面</string>
   </property>
  </action>
  <action name="chargeAction">
   <property name="text">
    <string>缴费窗口</string>
   </property>
  </action>
  <action name="infoAction">
   <property name="text">
    <string>信息管理</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
